docs/upgrade/money-class.md

```diff
@@ -0,0 +1,430 @@
+# Upgrade Instructions for Money Class
+
+This article describes upgrade instructions for [#821 new class for representing monetary values](https://github.com/shopsys/shopsys/pull/821).
+Upgrade instructions are in a separate article because there is a lot of instructions and we don't want to jam the [UPGRADE-v7.0.0.md](/docs/upgrade/UPGRADE-v7.0.0.md).
+Follow these instructions only if you upgrade from `v7.0.0-beta6` to `v7.0.0`.
+
+All monetary values (*prices, account balances, discount amounts, price limits etc.*) should be represented by an instance of the class `\Shopsys\FrameworkBundle\Component\Money\Money`.
+
+Before the upgrade, we recommend reading the article [How to Work with Money](/docs/introduction/how-to-work-with-money.md) which explains the concept in detail.
+
+When in doubt, you can take a look at [what was modified in `shopsys/project-base` during this change](https://github.com/shopsys/project-base/compare/cb6d02f335819aeff575dec01bda5b228263a2eb...c08cac7b55ebc46b43c2e988d36e2f122cbb4598#files_bucket).
+
+## Configuration
+Register the new Doctrine type in `doctrine.dbal.types` in your `app/config/packages/doctrine.yml`:
+```diff
+         password: "%database_password%"
+         charset: UTF8
+         server_version: "%database_server_version%"
++        types:
++            money: \Shopsys\FrameworkBundle\Component\Doctrine\MoneyType
+
+     orm:
+         auto_generate_proxy_classes: false
+```
+
+## Extended and Custom Entities
+If you have created your custom entities that store monetary value or added a new such column [by entity extension](/docs/extensibility/entity-extension.md), change your `decimal` Doctrine type to the new `money` type:
```

Comment:
Extremely important note™ ;-)

Double dots in the end of the line is quite disturbing for me, I see you are using it consistently, but I would prefer to omit it everywhere

Reply:
Well, it's supposed to mean "...like in this example right here:"

I can remove the colons from such imperatives if it you find it disturbing, but there are some cases in which ending the line with colon is definitely appropriate (*I'm always talking about the first line*):

>There are some cases in which you'll have to modify your templates:
>
>### Manipulation with money before rendering
>Instead of arithmetic operators, ...

>See all the changes that were needed in shopsys/project-base:
>
>- in @ShopsysShop/Front/Content/Product/detail.html.twig:
>  ```diff
>        content="{{ currencyCode(domain.id) }}"
>    ...
>  ```

>These 3 data fixture classes had to be changed in `shopsys/project-base`:
>- `\Shopsys\ShopBundle\DataFixtures\Demo\PaymentDataFixture.php`
>- `\Shopsys\ShopBundle\DataFixtures\Demo\ProductDataFixtureLoader.php`
>- `\Shopsys\ShopBundle\DataFixtures\Demo\TransportDataFixture.php`

Reply:
ok, keep them ;-)